aptos-core/consensus/src/pipeline/buffer_manager.rs

```diff
@@ -411,11 +411,6 @@ impl BufferManager {
                     }))
                     .await
                     .expect("Failed to send persist request");
-                if commit_proof.ledger_info().ends_epoch() {
```

Comment:
why delete these lines

Reply:
The buffer manager has been reset twice, once here and once in process_reset_request. process_reset_request is called when the epoch manager performs an epoch change. If this part is not removed, the block buffer manager will be released before the epoch change is done, and there will be hidden bugs